### PR TITLE
chapter3 4 5

### PR DESCRIPTION
**Pull request by muizzuddinyafiq**
ready

### CODE REVIEW

**Comment by muizzuddinyafiq** [comment]
ready

File: app/assets/stylesheets/static_pages.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the StaticPages controller here.
```
Comment (ghost) [inline app/assets/stylesheets/static_pages.scss]: delete this file

File: app/helpers/static_pages_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
```
Comment (ghost) [inline app/helpers/static_pages_helper.rb]: delete this file

File: app/views/layouts/_header.html.erb [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+<div class="container">
+<%= link_to "sample app", "#", id: "logo" %>
+<nav>
+<ul class="nav navbar-nav navbar-right">
+<li><%= link_to "Home", "#" %></li>
+<li><%= link_to "Help", "#" %></li>
+<li><%= link_to "Log in", "#" %></li>
+</ul>
+</nav>
+</div>
```
Comment (ghost) [inline app/views/layouts/_header.html.erb]: ident please

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+  </head>
+
+ <body>
+<%= render "layouts/header" %>
+<div class="container">
+<%= yield %>
```
Comment (ghost) [inline app/views/layouts/application.html.erb]: ident please

File: app/views/static_pages/help.html.erb [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+<h1>Help</h1>
+<p>
+Get help on the Ruby on Rails Tutorial at the
+<a href="http://www.railstutorial.org/help">Rails Tutorial
+help page</a>.
+To get help on this sample app, see the
+<a href="http://www.railstutorial.org/book"><em>Ruby on Rails
+Tutorial</em>
+book</a>.
```
Comment (ghost) [inline app/views/static_pages/help.html.erb]: ident please

File: config/database.yml [hunk 6]
```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
```
Comment (ghost) [inline config/database.yml]: please git ignore  this file

**Review by ghost — COMMENTED** [review]
cmt

File: config/database.yml (hunk 6 above)
Comment (muizzuddinyafiq) [reply config/database.yml]: how to do this?

**Comment by muizzuddinyafiq** [comment]
ready

**Comment by muizzuddinyafiq** [comment]
readfy

**Comment by muizzuddinyafiq** [comment]
ready

**Review by ghost — COMMENTED** [review]
please fix your conflict

**Comment by muizzuddinyafiq** [comment]
ready

**Review by ghost — COMMENTED** [review]
You have 2 commit, please make it 1

**Comment by muizzuddinyafiq** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
You can merge this pull request

**Comment by huongnt-2545** [comment]
merged

**Pull request merged**